nucypher/blockchain/eth/trackers/dkg.py

```diff
@@ -140,7 +140,7 @@ def _get_start_block_number(self) -> int:
         ):
             expected_start_block = w3.eth.get_block(expected_start_block.number - 1)
 
-        return expected_start_block.number
```

Comment:
Do we need to be careful here. Could this return -1 if the expected_start_block.number is 0 - might be possible in tests?